### ISSUE TITLE
关于calc_kg_loss中计算pos_score时的报错。

### ISSUE THREAD

**Issue by yyytiancai** [Userepic/KGAT-pytorch-master#2]
您好，有一个问题想请教一下。
当我运行到这一步的时候：
kg_batch_loss = model('calc_kg_loss', kg_batch_head, kg_batch_relation, kg_batch_pos_tail, kg_batch_neg_tail)
出现了报错。
然后我尝试在calc_kg_loss方法中，print出pos_score，定位到了问题出在计算pos_score这一步。

部分报错信息如下：
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [64,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [65,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [66,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [67,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [68,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [69,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [70,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [71,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [72,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [73,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [74,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [75,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [76,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [77,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [78,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [79,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/pytorch/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [13,0,0], thread: [80,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
。。。。。
File "train.py", line 137, in <module>
    kg_batch_neg_tail)
  File , line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File , line 190, in forward
    return self.calc_kg_loss(*input)
  File , line 168, in calc_kg_loss

前面的r_mul_h，r_embed，r_mul_pos_t，打印出来size都是（256，64）。
但是这一步计算torch.sum(torch.pow(r_mul_h + r_embed - r_mul_pos_t, 2), dim=1) 无法正常执行了，能否帮忙看一下问题出现在哪里呢？非常感谢！

**Comment by kangxiatao** [comment]
很抱歉，这个还是我去年的课题，一些细节问题我可能记不清楚了，可能帮不到你。

但是程序中应该是没有train.py这个文件的，你看是不是在你修改的过程中逻辑不对。你可以直接运行原始的程序看有没有错误，如果还有错误不排除调用的包版本不兼容问题（我印象中dgl建图这个包版本比较多，我遇到过因为版本不对而莫名其妙的问题）。
